test/components/settings/NonTandem.test.js

```diff
@@ -1,546 +0,0 @@
-/*
```

Comment:
Was this file deleted unintentionally? Looks like it was part of a temporary test removal and didn't get restored as part of the undo.